CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### citus v9.3.1 (Jun 17, 2020) ###
+
+* Adds support to partially push down tdigest aggregates
+
+* Fixes a crash that occurs inserting implicitly coerced constants
```

Comment:
occurs **when** inserting

I guess we'll not go over the character limit. Please check that as well